### DISCUSSION TITLE
Deploy PRs to Github Pages for Preview

### DISCUSSION THREAD

**Discussion by 5h3rr1ll** [KitaPlatzZentrale/kpz#110] [Ideas]
The idea is to have a GitHub action in place that deploys PR to the GitHub pages feature from GitHub.

For that, we always have a life preview of the latest changes, even before we merge it dev, and another developer doesn't need to check out the PR changes to see the changes.

Here are some resources that say how to do it:
- https://dev.to/dyarleniber/setting-up-a-ci-cd-workflow-on-github-actions-for-a-react-app-with-github-pages-and-codecov-4hnp
- https://github.com/gitname/react-gh-pages

**Comment by Darjusch** [comment]
From my point of view it looks like a nice to have feature but I don't think we really need that right now. 
I also can't asses how difficult it is to setup. 

If we get really rolling again and have a lot of changes incoming maybe it is worth it.
